proof/refine/ARM/ArchCSpace_R.thy

```diff
@@ -662,7 +690,7 @@ lemma ex_nonz_cap_not_global':
   apply clarsimp
   apply (drule orthD1, erule (1) subsetD)
   apply (subgoal_tac "s \<turnstile>' cteCap cte")
-   apply_trace (clarsimp simp: valid_cap'_def capRange_def capAligned_def
+   apply (fastforce simp: valid_cap'_def capRange_def capAligned_def
```

Comment:
Out of curiosity, here did this `apply_trace` come from? It doesn't seem to be in AARCH64's version but appears as part of the clobber?

Reply:
Artifact of the commit-split procedure: I noticed that I had added it on AARCH64 when debugging and left it in, so I fixed that and squashed it in. On ARM, it got copied over but then fixed later.